### PR TITLE
Updated readme file

### PR DESCRIPTION
**Pull request by shamshaw**
Updated readme file which describes data processing steps and cleans up language for others to recreate

### CODE REVIEW

File: 19_shamshaw/readme.md [hunk 1]
```diff
@@ -1,39 +1,54 @@
...
+This data visualization shows the occurence of streamflow drought events observed at USGS streamgages in the Upper Colorado River Basin. For the purposes of this visualization, a streamflow drought event is defined as the time period when streamflow drops below the 5th percentile of streamflows that have been observed on that date (utilizing a 30-day moving window). The source data for this plot is daily streamflow records downloaded from  [USGS NWIS](https://waterdata.usgs.gov/nwis/sw). 
 
-Run the targets pipeline to build the base visualizations. 
+## Data Preparation
 
-# Messaging
+To prepare the plot data file the following processing steps were performed:
 
-## Tweet 1
+ - Audit streamflow records to filter all available data to only data from streamgages where records between 1980 and 2020 have 5% or less missing data.
```
Comment (cnell-usgs) [inline 19_shamshaw/readme.md]: Maybe add a note here saying "This resulted in # gage locations in the Upper Colorado"

**Review by cnell-usgs — APPROVED** [review]
Just one small note, otherwise good to merge

**Comment by shamshaw** [comment]
@ceenell I should have merged this and done a new PR but realize now the changes I just pushed up are showing here now. So, if you want to take a look at the updated `_targets.R` file it should be much streamlined. We'll just need to update the path to the data if it is posted on S3 somewhere.

**Comment by shamshaw** [comment]
By the way, the plots that get exported out on my computer the color looks washed out compared to the versions from you. Is that a rendering on mac difference?

**Comment by cnell-usgs** [comment]
Huh, that's funny about the color. I have no idea why that would happen. Seems like it's just the 40 year chart though.

**Comment by cnell-usgs** [comment]
Merging and will add the data link

**Pull request merged**